starter_code/public/javascript/APIHandler.js

```diff
@@ -1,25 +1,125 @@
 class APIHandler {
-  constructor (baseUrl) {
+  constructor(baseUrl) {
     this.BASE_URL = baseUrl;
   }
 
```

Comment:
you could have created a function to make the API calls and then make smaller functions for the CRUDs(like this):
```
callToAPI (url, method, callback, data) {
    url = `${this.BASE_URL}${url}`;
    $.ajax({
      method: method,
      url: url,
      data: data,
      success: callback,
      error: this.errorHandler,
      datatype: "json",
    })
  }
```